src/components/AppCode.vue

```diff
@@ -70,8 +86,14 @@ export default {
         selection.addRange(range);
       }
 
-      document.execCommand("copy");
-      alert("copied!");
+      let copied = document.execCommand("copy");
+
+      if (copied) {
+        this.codeWasCopied = true;
+        setTimeout(() => {
```

Comment:
Not sure about this change: why does this need to wait 2 seconds for a11y purposes?

Reply:
@sdras I'm not the author of the PR and I don't know if it's an a11y improvement, but the text temporary text change makes sense to me. 

This way the user knows it was copied and have the opportunity to see the old "Copy Me" text 2 seconds later. 

The alert() function just freezes the page.

Reply:
Sorry, I wasn't clear, yeah I understand the functionality, I was wondering why the wait needed to be 2 seconds in particular, if there's a benefit for a11y purposes or if it was an arbitrary amount of time :)

Reply:
2 seconds is just an arbitrary amount of time for what's essentially a 'flash message' to the user. Only necessary if they intend to copy multiple times.

I don't know if there's any research on the ideal timeout for this sort of message!

Reply:
Great, thank you for clarifying :)